metrics/grafana/tikv_details.json

```diff
@@ -13511,8 +13612,25 @@
             "x": 0,
             "y": 109
           },
+          "heatmap": {},
+          "hideZeroBuckets": true,
+          "highlightCards": true,
           "id": 7236,
+          "legend": {
+            "show": false
+          },
           "links": [],
+          "repeat": null,
+          "repeatDirection": null,
+          "repeatedByRow": true,
+          "reverseYBuckets": false,
+          "scopedVars": {
```

Comment:
Why change it?

Reply:
I have no idea why it's changed. I will revert it.